packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -162,6 +162,72 @@ When the `smsRole` property is specified, the `smsRoleExternalId` may also be sp
 assume role policy should be configured to accept this value as the ExternalId. Learn more about [ExternalId
 here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html).
 
+User pools can be configured to enable MFA. It can either be turned off, set to optional or made required. Setting MFA
+to optional means that individual users can choose to enable it.
+Additionally, the MFA code can be sent either via SMS text message or via a time-based software token.
+See the [documentation on MFA](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-mfa.html) to
+learn more.
+
+The following code snippet marks MFA for the user pool as required. This means that all users are required to
+configure an MFA token and use it for sign in. It also allows for the users to use both SMS based MFA, as well,
+time-based one time password.
+
+```ts
+new UserPool(this, 'myuserpool', {
+  // ...
+  mfaEnforcement: MfaEnforcement.REQUIRED,
+  mfaToken: {
```

Comment:
Maybe? Seems like this is the terminology they use the [AWS docs](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-mfa.html#user-pool-configuring-mfa)

```suggestion
  mfaSecondFactor: {
```